app/controllers/clearance/passwords_controller.rb

```diff
@@ -45,7 +45,7 @@ def new
   def update
     @user = find_user_for_update
 
-    if @user.update_password password_reset_params
+    if @user.update_password(password_reset_params)
```

Comment:
Since `password_reset_params` could return nil, would calling `@user.update_password(nil)` result in a user's password being set to `nil`?